controllers/config/grafanaIni.go

```diff
@@ -702,6 +702,10 @@ func (i *GrafanaIni) cfgUnifiedAlerting(config map[string][]string) map[string][
 	items = appendStr(items, "evaluation_timeout", i.cfg.UnifiedAlerting.EvaluationTimeout)
 	items = appendInt(items, "max_attempts", i.cfg.UnifiedAlerting.MaxAttempts)
 	items = appendStr(items, "min_interval", i.cfg.UnifiedAlerting.MinInterval)
+
+	items = appendStr(items, "ha_advertise_address", "${POD_IP}:9094")
```

Comment:
is this always hardcoded?

Reply:
Well it would need the pod IP I believe, but if you're referring to the port I believe that can be made configurable if we want to add that field to the Grafana CRD.